Frontend/ui-customer/src/Flow.purs

```diff
@@ -2168,6 +2168,11 @@ homeScreenFlow = do
       modifyScreenState $ ReferralScreenStateType (\referralScreen -> referralScreen { referralType = referralType, referralCode = if any (_ == referralCode) [ "__failed", "" ] then "" else referralCode })
       referralScreenFlow
     ON_CALL state callType exophoneNumber -> do
+      case callType of
+        ANONYMOUS_CALLER -> do
+          (_ :: (Either ErrorResponse CallOnClickResp)) <- lift $ lift $ HelpersAPI.callApi (CallOnClickReq state.data.driverInfoCardState.rideId )
+          pure unit
+        _ -> pure unit
       (APISuccessResp res) <- Remote.onCallBT (Remote.makeOnCallReq state.data.driverInfoCardState.rideId (show callType) exophoneNumber)
```

Comment:
is it expected to call both exotelCallEvent and normalCallEvent API simultaneously ? and why?

Reply:
![Uploading image.png…]()